viz_scripts/bin/generate_plots.py

```diff
@@ -3,6 +3,13 @@
 import nbparameterise as nbp
 import argparse
 import arrow
+import json
+import requests
+
+
+# Configuration settings to use for all generated plots by this instance
+# This could also be specified as a parser argument, if we want to generate plots for all programs from one instance
+STUDY_CONFIG = "stage-program"
```

Comment:
This is fine for now, but we need to think about how we can get it in the docker container.
We might need to read this from an environment variable in that case since we can pass in environment variables to containers.